test/test_view/test_bloomberg.py

```diff
@@ -65,3 +65,13 @@ def test_login_post_not_authenticated(self, mock_one_login):
         mock_one_login.return_value = mock_auth
         res = self.app.post('/bloomberg/login')
         assert res.status_code == 302, res.status_code
+
+    @with_context
+    @patch('pybossa.view.bloomberg.OneLogin_Saml2_Auth', autospec=True)
+    def test_login_create_account_fail(self, mock_one_login):
```

Comment:
Could we include a test for `test_login_create_account_success` with a mock api call? Just to ensure the correct code-paths are covered. It doesn't have to actually call the real api (you can "patch" the API call). Thanks.